include/sound/sof/ipc4/header.h

```diff
@@ -383,7 +383,7 @@ struct sof_ipc4_fw_version {
 	uint16_t minor;
 	uint16_t hotfix;
 	uint16_t build;
-};
+} __packed;
```

Comment:
@ujfalusi unrelated to this PR. with SOF+IPC4, FW version will include only the major/minor/patch right?